src/rules/whitespaceRule.ts

```diff
@@ -50,14 +51,17 @@ export class Rule extends Lint.Rules.AbstractRule {
             * \`"check-type"\` checks for whitespace before a variable type specification.
             * \`"check-typecast"\` checks for whitespace between a typecast and its target.
             * \`"check-type-operator"\` checks for whitespace between type operators \`|\` and \`&\`.
-            * \`"check-preblock"\` checks for whitespace before the opening brace of a block`,
+            * \`"check-preblock"\` checks for whitespace before the opening brace of a block.
+            * \`"check-postblock-prekeyword"\` checks for whitespace between the closing brace of a block
+            and the following keyword (\`} else\` / \`} catch\` / \`} while\`).`,
```

Comment:
what's the difference to `one-line` with `"check-whitespace"`?

Reply:
I reviewed the `one-line` rule code and its test case. Didn't find that rule solve the situation of this PR. And I tried to add `one-line` with all its options to my test file, the linter didn't report error of missing whitespace between closing bracket and `else`

Reply:
#3200 includes the check for whitespace before `else`.
The only thing missing is whitespace before `while` in `do {...} while (...);` loops. But that could easily be added.

Reply:
So is my pr going to be merged?

Reply:
I have no preference into which rule the functionality should go. However, we generally try to avoid duplication.
I'll leave that decision to @adidahiya

Reply:
@yubaoquan do you generally use `one-line` with `"check-whitespace"` in your lint config? if so, I think we should just add the additional check @ajafff suggested to `one-line` instead of a new option here.